test/e2e/sanity_test.go

```diff
@@ -26,3 +32,44 @@ func TestOperatorLabel(t *testing.T) {
 		t.Errorf("Expected node selector 'linux', not '%s'", osSelector)
 	}
 }
+
+func TestNoDegraded(t *testing.T) {
```

Comment:
This test should check that the daemon has done its sync atleast once before checking if it degraded.

Reply:
Hmm...like wait for the `state=Done` annotation on the node?

Reply:
That would work if looking at it from the end of the run. You could also watch to see `Working` and know that it sync'd.